guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -126,7 +126,7 @@ The `<action>` instruction is deprecated. If the method implementation allows, u
 
 Calls public methods on the block API.
 
-**Details:** Used to set up the execution of a certain method of the block during block generation; the `<action>` node must be located in the scope of the `<block>` node.</p>
+**Details:** Used to set up the execution of a certain method of the block during block generation; the `<action>` node must be located in the scope of the `<block>` node.
```

Comment:
v2.1 of the file still has the <p> and </p> html tags. Can you remove those tags as well?